tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -118,12 +119,30 @@ stages:
                   - checkout: none
                   - download: current
                     artifact: release-artifacts
+                  - pwsh: |
+                      $latestVersion = (azcopy list "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp/") -replace '[\/;].*' | Get-Unique | sort {[version] $_} -Descending | Select-Object -First 1
+                      $major, $minor, $build = $latestVersion.Split('.')
+                      $nextVersion = [version]::new($major, 1 + $minor, $build).ToString()
+                      Write-Host "##vso[task.setvariable variable=APIVIEW_PARSER_VERSION]$nextVersion"
+                    displayName: Determine Next ApiView Parser Version
+                  - task: AzurePowerShell@5
+                    displayName: Publish Public ApiView Parser
+                    inputs:
+                      azureSubscription: 'Azure SDK Artifacts'
+                      ScriptType: 'InlineScript'
+                      azurePowerShellVersion: LatestVersion 
+                      pwsh: true
+                      Inline: |
+                        azcopy copy "$(Pipeline.Workspace)/release-artifacts/*" "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp/$(APIVIEW_PARSER_VERSION)" --recursive
+                    env:
+                      AZCOPY_AUTO_LOGIN_TYPE: 'PSCRED'
                   - task: UniversalPackages@0
```

Comment:
If we plan to consume using the bloc we should probably stop publishing here.

Reply:
so are we only publishing to public then?

Reply:
I don't see any reason we need it on both.

Reply:
ok make sense